src/components/Layout/Layout.tsx

```diff
@@ -4,7 +4,7 @@ import Header from './header/index'
 import Breadcrumbs from './breadcrumbs/Breadcrumbs'
 import Footer from './footer/index'
 
-const Layout = () => {
+function Layout() {
```

Comment:
why did you change type of function here?

Reply:
Done